backend-services/autoignore/requirements.txt

```diff
@@ -1,4 +1,4 @@
-artemis-utils==1.0.5
+#artemis-utils==1.0.5
```

Comment:
TODO and for all others: reinstate this to new version 1.0.6 after approval

Reply:
upgraded since even with symlink utils is outside the current build context. Will test  the new version directly.